src/Shout/SHRBTextStyler.class.st

```diff
@@ -3,8 +3,7 @@ I'm the one who visits node to coloring the code
 "
 Class {
 	#name : #SHRBTextStyler,
-	#superclass : #SHTextStyler,
-	#traits : 'TRBProgramNodeVisitor',
+	#superclass : #RBProgramNodeVisitor,
 	#classTraits : 'TRBProgramNodeVisitor classTrait',
```

Comment:
For the class side, this is a bug I had often when working on Moose. I reported it but I never had the time to check the origin and fix it :(

And no, the trait is not needed anymore and I wonder what to do with it. 
- Keep it for external users that might want a visitor that does not inherit from the actual visitor
- Deprecate it and remove it in P13
- Remove it (I prefer option 2)

I would vote to deprecate it and remove it in P13 because the actual visitor cannot use the Trait IIRC because it is loaded before the traits are in the image. And it causes a big duplication that can be hard to maintain since we are improving the AST a lot recently.